pkg/version/version.go

```diff
@@ -28,7 +28,7 @@ import (
 
 var (
 	// SupportedVersionsRange holds all supported temporal versions.
-	SupportedVersionsRange  = mustNewConstraint(">= 1.14.0 < 1.26.0")
+	SupportedVersionsRange  = mustNewConstraint(">= 1.14.0 < 1.29.0")
 	ForbiddenBrokenReleases = []*Version{
```

Comment:
Could you please add 1.27.0 as forbidden? See: https://github.com/temporalio/temporal/releases/tag/v1.27.0